atm/src/forcing.F90

```diff
@@ -105,12 +107,21 @@ subroutine forcing_init_fields(self, fields, forcing_date, &
             num_land_fields = num_land_fields + 1
         endif
 
+        ! simple scale_constant
+        call self%core%get(fp, "scale_constant", scale_constant, scale_constant_found)
+
+        call assert( .not. (scaling_found .and. scale_constant_found), &
```

Comment:
is there any reason not to allow both scalings?

Reply:
The code in `field.F90` works if both are chosen but I thought it would be better to restrict things to one method for the moment.